index.html

```diff
@@ -1,125 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="./images/favicon-32x32.png"
-    />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@500;600&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="/css/style.css" />
-    <script src="/index.js" type="module" async></script>
-    <title>Frontend Mentor | Interactive card details form</title>
-  </head>
-  <body>
-    <main>
-      <section class="card-section">
-        <div class="cards-mobile">
-          <div class="card-back card">
-            <span class="security-number">000</span>
-          </div>
-          <div class="card-front card">
-            <img
-              class="card-logo"
-              src="/images/card-logo.svg"
-              alt="Card logo."
-            />
-            <p class="card-number">0000 0000 0000 0000</p>
-            <div class="card-info">
-              <span class="card-name"></span>
-              <span class="expire-date">
-                <span class="exp-month">00</span>
-                /
-                <span class="exp-year">00</span>
-              </span>
-            </div>
-          </div>
-        </div>
-        <div class="cards-desktop">
-          <div class="card-front card">
-            <img
-              class="card-logo"
-              src="/images/card-logo.svg"
-              alt="Card logo."
-            />
-            <p class="card-number">0000 0000 0000 0000</p>
-            <div class="card-info">
-              <span class="card-name">JANE APPLESEED</span>
-              <span class="expire-date">
-                <span class="exp-month">00</span>
```

Comment:
**question:** Is there any reason to wrap this in the span? You don't even use this class to styling

Reply:
Yes, there is because while styling div.card-info I use flex to position this and when I don't wrap it in that span it's just looks bad. Firstly I had other idea for this element, but this is its final form. I could change it into a div, but I don't think that matters that much. Although I can remove the class it's useless.